src/main/java/Duke/Duke.java

```diff
@@ -0,0 +1,232 @@
+                writer.write("\r\n");
+            }
+            writer.close();
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+    }
+
+    private static void addItemToItemBook(String commandType, String commandData) {
+        allItems[count][0] = commandType;
+        allItems[count][1] = UNDO;
+        allItems[count][2] = commandData;
+        count++;
+        // writeFile(commandType+"-"+UNDO+"-"+commandData);
+    }
+
+    private static void showWelcomeMessage() {
+        showToUser(DIVIDER, DIVIDER, VERSION, LOGO, GREETING, DIVIDER);
+    }
+
+    private static void showToUser(String... message) {
+        for (String m : message) {
+            System.out.println(m);
+        }
+    }
+
+    private static String getUserInput() {
+        System.out.print("Enter command: ");
+        String inputLine = SCANNER.nextLine();
+        // silently consume all blank and comment lines
+        while (inputLine.trim().isEmpty() || inputLine.trim().charAt(0) == INPUT_COMMENT_MARKER) {
+            inputLine = SCANNER.nextLine();
+        }
+        return inputLine;
+    }
+
+    private static void echoUserCommand(String userCommand) {
+        showToUser("[Command entered:" + userCommand + "]", DIVIDER);
+    }
+
+    private static int executeCommand(String userInputString) {
+        final String[] commandTypeAndParams = splitCommandWordAndArgs(userInputString);
+        final String commandType = commandTypeAndParams[0];
+        final String commandArgs = commandTypeAndParams[1];
+        switch (commandType) {
+        case COMMAND_TODO_WORD:
+            addTodoItem(commandArgs);
+            return 1;
+        case COMMAND_EVENT_WORD:
+            addEventItem(commandArgs);
+            return 1;
+        case COMMAND_DEADLINE_WORD:
+            addDeadlineItem(commandArgs);
+            return 1;
+        case COMMAND_LIST_WORD:
+            showListItem();
+            return 1;
+        case COMMAND_DONE_WORD:
+            doneItem(commandArgs);
+            return 1;
+        case COMMAND_END_WORD:
+            endSystem();
+            return 0;
+        case COMMAND_DELETE_WORD:
+            deleteItem(commandArgs);
+            return 1;
+        default:
+            showError();
+            return 1;
+        }
+    }
+
+    private static String[] splitCommandWordAndArgs(String rawUserInput) {
+        final String[] split = rawUserInput.trim().split("\\s+", 2);
+        return split.length == 2 ? split : new String[] { split[0] , "" }; // else case: no parameters
+    }
+
+
+    public static void showListItem(){
+        for (int i = 0; i<count; i++){
+            int index = i+1;
+            showToUser(index+". "+LEFTPAR+allItems[i][0]+RIGHTPAR+LEFTPAR+allItems[i][1]+RIGHTPAR+allItems[i][2]);}
+    }
+
+    public static void showError(){
+        showToUser(ERROR_MESSAGE,DIVIDER);
+    }
+
+    public static void addDeadlineItem(String commandArgs){
```

Comment:
Code organisation is important; you could perhaps better organise your code segregating related methods into their own classes